src/js/xr/src/SceneManagerXR.js

```diff
@@ -304,9 +306,12 @@ const SceneContent = connect(
     }, [])
     const helpVoicer = useMemo(() => {
       let voicer = new Voicer(cameraAudioListener, 10, null, {
-        releaseTime: 0.2
+        releaseTime: 0.2,
```

Comment:
I'd like to try shorter releaseTime here, 0.1 or 0.05 to see how low we can get it w/o clicks